scripts/list.formulae

```diff
@@ -2,14 +2,19 @@
 
 set -xe
 
+# OS we're testing formulae on
+os="${1:-DUMMY_OS_VALUE}"
+
 # Let's see where we're running
 myroot=$(cd "$(dirname "$0")"; pwd)
 cd "$(dirname "${myroot}")"
 
 # List all formulae ending in .rb in this folder
 output=""
 for f in *.rb; do
-    output+="$(basename "${f}" .rb) "
+    if ! grep KAUST_SKIP "${f}" | grep "${os}"; then
```

Comment:
@antoniokaust  can you please explain to me defining the "os" variable and using it here?

Reply:
I looking for the line `KAUST_SKIP` then on that line I'm looking for the OS you want to skip on , e.g. `centos:6`.